plugins/CoreAdminHome/Commands/DeleteLogsData.php

```diff
@@ -100,6 +100,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
         if ($input->getOption('optimize-tables')) {
             $this->optimizeTables($output);
         }
+
+        return 0;
```

Comment:
Maybe can we use SymfonyCommand::SUCCESS and SymfonyCommand::FAILURE here ?

Theses [constants are now available in matomo](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Console/Command/Command.php#L33) project